jehy/telegram-test-api
# `request` was added as a dependency. Version was bumped to 1.0.3

## How to reproduce the problem

1. Remove your `node_modules/`
1. Install dependencies the way it does `npm` when your package is some other project's dependency:
   ```sh
   npm install --only=production
   ```
1. Try to run the server:
   ```sh
   node bin/start.js  
   ```
1. You'll see:
   ```
   ###
   ### The "request" library is not installed automatically anymore.
   ### But required by "request-promise".
   ###
   ### npm install request --save
   ###
   
   / ... /telegram-test-api/node_modules/request-promise/lib/rp.js:23
       throw err;
       ^
   
   Error: Cannot find module 'request'
   ...
   ```

## Solution

I've added `request` to dependencies' list. You're able to make sure that you need to specify `request` in your package.json at `request-promise`'s README:

> request became a peer dependency. Thus make sure that request is installed into your project as a direct dependency. (npm install --save request)

[source](https://github.com/request/request-promise#migration-from-v3-to-v4)
